bpython/curtsiesfrontend/replpainter.py

```diff
@@ -161,6 +161,12 @@ def formatted_argspec(funcprops, arg_pos, columns, config):
 
 
 def formatted_docstring(docstring, columns, config):
+    if isinstance(docstring, bytes):
+        docstring = docstring.decode('utf8')
+    elif isinstance(docstring, str if py3 else unicode):
+        pass
+    else:
+        return []
```

Comment:
Is the `elif` and `else` really necessary. Or in other words: does the `elif` really cover all valid cases?